package.json

```diff
@@ -23,45 +23,45 @@
   "dependencies": {
     "@koa/router": "^10.1.1",
     "@octokit/rest": "^18.12.0",
-    "@polkadot/api": "^7.15.1",
-    "@polkadot/keyring": "^8.7.1",
-    "@types/cron": "^1.7.3",
+    "@polkadot/api": "^8.8.1",
```

Comment:
are you sure you are ready for this major ?

Reply:
I think it should be fine. There haven't really been breaking changes in a while

Reply:
FYI be is now crushing, probably because of this